server/seasons.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const db = require('APP/db');
+const Season = db.model('season');
+const router = require('express').Router();
+
+// Create:
+// add a season
+
+router.post('/admin/new', (req, res, next) => {
```

Comment:
Same here, we should try to keep it consistent without the admin specification in the route because we will be using the auth session to verify if the user is logged in and if he/she is an Admin